tracee-rules/Readme.md

```diff
@@ -33,6 +33,8 @@ This will:
 
 When a detection is made by any of the signatures, it will be printed to stdout. Using the `--webhook` flag you can post detections into an HTTP endpoint that can further relay the detection.
 
+You can also use a custom template (or use a pre-supplied one) to further tune your webhook detection output. This can be passed in via the `--webhook-template` flag.
```

Comment:
We should document what it means to supply a custom template

Reply:
Thanks - I expanded on it https://github.com/aquasecurity/tracee/pull/582/commits/035e838ee01702bd8e96baaf991d67ec2fe1f373